app/templates/components/modals/event-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete this event?'}}
+  <div class="muted small text">
+    {{t 'Deleting the event will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action 'submit' on='submit' preventDefault=true}}>
```

Comment:
No. This would throw an error now if the user submits the form by pressing the enter key when typing into the input field. 

That's why do what I said 

`{{action (option anyRandomVariableThatMayNotExist on='submit' preventDefault=true}}`